source/s3_auto_ranged_get.c

```diff
@@ -139,6 +166,22 @@ static bool s_s3_auto_ranged_get_update(
             }
         }
 
+        if (auto_ranged_get->synced_data.get_without_range) {
```

Comment:
Previously, I thought it can help if we want to have a get request without range from the beginning of the process. But, yeah, seems like it should just for the special error handling.